flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/ProcTimeBoundedRangeOver.scala

```diff
@@ -123,6 +124,8 @@ class ProcTimeBoundedRangeOver(
       return
     }
 
+    out.asInstanceOf[TimestampedCollector[_]].eraseTimestamp()
```

Comment:
Do we really need these erasing steps in every function? Shouldn't a rowtime operator overwrite it anyway?